WBooks/Library/Details/BookInfo/BookInfoViewController.swift

```diff
@@ -0,0 +1,92 @@
+//
+//  BookInfoViewController.swift
+//  WBooks
+//
+//  Created by ana.mancuso on 17/10/2022.
+//
+
+import UIKit
+
+class BookInfoViewController: UIViewController {
+    
+    //MARK: Properties
+    
+    private lazy var bookView = BookInfoView()
+    private let bookModel: BookInfoViewModel
+    
+    //MARK: Initialization
+    
+    init (bookModel: BookInfoViewModel) {
+        self.bookModel = bookModel
+        super.init(nibName: nil, bundle: nil)
+    }
+    
+    required init? (coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        configBook()
+    }
+    
+    override func loadView() {
+        view = bookView
+    }
+    
+    func configBook() {
+        setupData(library: bookModel.bookDetails)
+        bookView.rentButton.addTarget(self, action:#selector(rentButton), for: .touchUpInside)
+        bookView.rentButton.applyGradient(colors: ((bookView.availabilityBook.text == "Available") ? [UIColor.lightSeaGreen.cgColor, UIColor.summerSky.cgColor,  UIColor.mediumTurquoise.cgColor] : [UIColor.gainsboro.cgColor]))
+        bookView.addButton.addTarget(self, action:#selector(addButton), for: .touchUpInside)
+        bookView.availabilityBook.textColor = (bookView.availabilityBook.text == "Available") ? .sushi : .persianRed
+    }
+    
+    //MARK: Action
+    
+    @objc func addButton() {
```

Comment:
If for the moment the function is not going to be used, or for some reason it does not have anything, you should leave it like this in a line

`@objc func addButton() { }`